ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Plugins/tasks/nagios_Debian.yml

```diff
@@ -0,0 +1,35 @@
+---
+#################################################################################################
+# AdoptOpenJDK - Ansible Playbook to install Nagios plugins on Debian on x86 hardware #
+#################################################################################################
+
+###########################################################################
+# License Information:                                                    #
+# Nagios core and its plugins are lincesed under GPL                      #
+# For more information please see: https://www.nagios.com/legal/licenses/ #
+###########################################################################
+
+########################################
+# Install Nagios dependencies packages #
+########################################
+- name: Install Nagios plugins
+  apt: pkg={{ item }} state=latest update_cache=yes
+  with_items:
+    - fping
+    - gcc
+    - nagios-plugins
+    - nagios-plugins-common
+    - perl
+    - qstat
+  tags: [nagios_plugins, adoptopenjdk]
+
+##########
+# Layout #
+##########
+- name: Creates Nagios folder
+  file: path=/usr/local/nagios/ state=directory mode=0755 owner=nagios
+  tags: [nagios_plugins, adoptopenjdk]
```

Comment:
What about moving the tags to the role declaration instead of repeating them here over and over again?